alacritty/src/window.rs

```diff
@@ -361,6 +366,27 @@ impl Window {
         self.window().wayland_display()
     }
 
+    #[cfg(not(any(target_os = "macos", target_os = "windows")))]
+    pub fn set_wayland_theme(&mut self, config: &Config) {
+        let hovered_close_icon = config.colors.normal().red;
+        let hovered_maximize_icon = config.colors.normal().green;
+        let hovered_minimize_icon = config.colors.normal().yellow;
+        let foreground = config.colors.primary.foreground;
+        let background = config.colors.primary.background;
+        let dim_foreground =
+            config.colors.primary.dim_foreground.unwrap_or_else(|| foreground * 0.66);
```

Comment:
Not a fan of this, since we duplicate the dimming code here.

It might be better to pass `&config.colors` and `term.colors` so we can do it properly?

Reply:
You don't have access to terminal colors and I don't really like adding an option to get them, maybe we should keep this simple for now?

Reply:
I'm fine with keeping it simple, I'm not fine with keeping in bug prone.

If we wanted to keep it simple, we'd have to remove the dimming.

Why do we not have access to the terminal colors, because this is executed right at startup?

Reply:
I mean they are `private` right now for us. We do have access for them, but it's about adding `pub` or methods to get them. This is not about error prone or not, let's say I've just dimmed color by some value if it wasn't available. I could remove use of dim and just use `0.64 * foreground`.

I feel like we can use dark black though.

Reply:
> This is not about error prone or not, let's say I've just dimmed color by some value if it wasn't available. I could remove use of dim and just use 0.64 * foreground.

This absolutely is about introducing bugs. Because if I change the dim colors in our color code, but forget to update the wayland theme, that's a bug.

> I feel like we can use dark black though.

No. That doesn't work for light themes.

Reply:
Oh yeah, we can use just `white`.